src/js/index.js

```diff
@@ -171,44 +221,37 @@ document.addEventListener("DOMContentLoaded", () => {
 
 async function updateContributorsTable() {
 
-    const contributors = await readContract({
+    const contributionsMetadata = await readContract({
         address: contractAddress,
         abi: contractABI,
-        functionName: 'contributionsByAddress',
+        functionName: 'getAllContributions',
         chainId: 5,
     });
 
-    return;
-
-    // Assuming the smart contract has a method 'getContributors' to fetch the array of contributors
-    myContract.methods.contributors().call()
-        .then((contributors) => {
-            // Get the table body
-            const tableBody = document.getElementById('contributorsTable');
-
-            // Clear the table body
-            tableBody.innerHTML = "";
-
-            // Loop through the contributors and append a row for each
-            contributors.forEach((contributorAddress, index) => {
-                // Create a new row and cells
-                let row = document.createElement('tr');
-                let th = document.createElement('th');
-                let td = document.createElement('td');
-
-                // Set the cell contents
-                th.textContent = index + 1; // Add 1 to index to start counting from 1
-                td.textContent = contributorAddress;
-
-                // Add the cells to the row
-                row.appendChild(th);
-                row.appendChild(td);
-
-                // Add the row to the table body
-                tableBody.appendChild(row);
-            });
-        })
-        .catch((error) => console.error);
-}
+    let contributionsByAddress = getContributionsByAddress(contributionsMetadata)
+    let leaders = getTopContributions(contributionsByAddress)
+
+    // Get the table body
+    const leaderboardTable = document.getElementById('leaderboard-table');
+    const leaderboardTableBody = leaderboardTable.childNodes[3]
+    const leaderRows = leaderboardTableBody.getElementsByTagName("tr");
```

Comment:
Getting different results for `leaderboard-table` when using development build vs. prod build.

When running `npm run devserver`, I get `<table />` with 5 child nodes:
![image](https://github.com/JustinHolmesMusic/VowelSoundsAlbumDrop/assets/12272640/7efdf20e-8d3b-426e-943f-5a9da3bc98a9)

The production build has the table with ordinary `2` child nodes:
![image](https://github.com/JustinHolmesMusic/VowelSoundsAlbumDrop/assets/12272640/081aeada-cb0a-4b6e-a5c2-1e2c0a062518)

So getting to the body with `index=3` won't work.
It is also broken on the currently deployed version.

Reply:
Whew... Not sure why that is.

We probably just need to operate on the ID of a lower-nested node.

If it works for you on devserver., then I think let's still merge this and work from there.

Reply:
Ok, creating new issue for that.